upgrade/Stage_2.md

```diff
@@ -19,36 +19,6 @@
 
 ## Stage 2.2
 
-1. Make sure that not all pods of `ingressgateway-hmn` or `spire-server` are running on the same worker node.
```

Comment:
this is removed because we added antiAffinity in 1.2 so we no long need this in 1.2 -> 1.3 upgrade